server/Makefile

```diff
@@ -1,8 +1,10 @@
 VERSION ?= "dev"
 TRACETEST_ENV ?= "dev"
+POKE_API?= "dev"
 GO_LDFLAGS := $(shell echo \
 		-X "'github.com/kubeshop/tracetest/server/app.Version=$(VERSION)'" \
 		-X "'github.com/kubeshop/tracetest/server/app.Env=$(TRACETEST_ENV)'" \
+		-X "'github.com/kubeshop/tracetest/server/app.PokeApi=$(POKE_API)'" \
```

Comment:
Don't forget to update the pipelines:

- [ ] `.github/workflows/deploy-main.yml#L65`
- [ ] `.github/workflows/pull-request.yml#L126`
- [ ] `.github/workflows/release-version.yml#L68`
- [ ] `.github/workflows/release-version.yml#L199`

Reply:
I dont think we need to add this to the deploys. Frontend code already have a default value, so if the PokeAPI is a empty string it should use the default value. We only need to change the poke API URL when running within the docker compose configuration

Reply:
It's not that its necessary, but it's a way to explicitly set the build args everywhere, so it's explicit how to change things